linera-chain/src/unit_tests/chain_tests.rs

```diff
@@ -155,26 +147,27 @@ async fn test_block_size_limit() -> anyhow::Result<()> {
     // The size of the executed valid block below.
     let maximum_block_size = 260;
 
-    let mut config = env.make_open_chain_config();
-    config.committees.insert(
-        Epoch(0),
-        bcs::to_bytes(&Committee::new(
-            BTreeMap::from([(
-                ValidatorPublicKey::test_key(1),
-                ValidatorState {
-                    network_address: ValidatorPublicKey::test_key(1).to_string(),
-                    votes: 1,
-                    account_public_key: AccountPublicKey::test_key(1),
-                },
-            )]),
-            ResourceControlPolicy {
-                maximum_block_size,
-                ..ResourceControlPolicy::default()
+    let committee = Committee::new(
```

Comment:
This committee is created and stored three times in this file. Let's deduplicate that.

Or should it even be part of `make_child_chain_description_with_config`?